sdk/client.go

```diff
@@ -33,19 +34,29 @@ type Options struct {
 	Sort   string
 }
 
-func (h *Headers) Add(req *http.Request) {
+type PatchOperation struct {
+	Op    string
+	Path  string
+	Value interface{}
+}
+
+func (h *Headers) Add(req *http.Request) error {
 	if h == nil {
-		return
+		return nil
```

Comment:
worth adding log info for missing header

Reply:
ok, done